### PR TITLE
Doctest corrections for Bio.AlignIO.MauveIO, Bio.File and Bio.Medline

### PR DESCRIPTION
**Pull request by MarkusPiotrowski**
This pull request addresses issue #1902 ('Failing doctests').

- [x] I hereby agree to dual licence this and any previous contributions under both
the _Biopython License Agreement_ **AND** the _BSD 3-Clause License_.

- [x] I have read the ``CONTRIBUTING.rst`` file and understand that AppVeyor and
TravisCI will be used to confirm the Biopython unit tests and ``flake8`` style
checks pass with these changes.

- [x] I have added my name to the alphabetical contributors listings in the files
``NEWS.rst`` and ``CONTRIB.rst`` as part of this pull request, am listed
already, or do not wish to be listed. (*This acknowledgement is optional.*)

### CODE REVIEW

File: Bio/AlignIO/MauveIO.py [hunk 1]
```diff
@@ -46,27 +46,32 @@
 would probably load this using the Bio.AlignIO.parse() function:
 
     >>> from Bio import AlignIO
-    >>> align = AlignIO.parse("Mauve/simple.xmfa", "mauve")
+    >>> align = AlignIO.parse("Mauve/simple_short.xmfa", "mauve")
     >>> alignments = list(align)
     >>> for aln in alignments:
-    ...     print(align)
+    ...     print(aln)
```
Comment (peterjc) [inline Bio/AlignIO/MauveIO.py]: Would ideally include another line here ``...`` as you would see in the Python terminal to close the for loop.

**Comment by peterjc** [comment]
Looks good, a tiny tweak suggested, but feel free to merge as is. Thanks!

File: Bio/AlignIO/MauveIO.py (hunk 1 above)
Comment (MarkusPiotrowski) [reply Bio/AlignIO/MauveIO.py]: Yes, I did it in other places, but when the tests passed I forgot about it. Will do.

**Comment by MarkusPiotrowski** [comment]
Again a single 'division by zero' error from one of the `Graphics` tests.

**Comment by peterjc** [comment]
The graphics test is stochastic, known annoyance - logged as #2040 and tagged as a good first issue for a new contributor.

**Comment by MarkusPiotrowski** [comment]
I know, I had this already last week, bad luck... Can you restart the test on Travis-CI?

**Comment by peterjc** [comment]
I wrongly assumed you could restart TravisCI, given it is linked to the GitHub accounts. I may need to look at the settings...

**Comment by MarkusPiotrowski** [comment]
Oh no, now AppVeyor stopped without an error...

**Comment by peterjc** [comment]
Admin privileges used to merge it anyway, AppVeyor had timed out on a download.

**Pull request merged**